components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/ExternalIdPConfig.java

```diff
@@ -249,6 +250,15 @@ public boolean isAssociateLocalUserEnabled() {
         return justInTimeProConfig != null && justInTimeProConfig.isAssociateLocalUserEnabled();
     }
 
+    public String getSyncAttributesMethod() {
+
+        String method = FrameworkConstants.SYNC_ALL;
+        if (justInTimeProConfig != null && justInTimeProConfig.getSyncAttributeMethod() != null) {
```

Comment:
Shall we add a blank check from StringUtils here ? justInTimeProConfig.getSyncAttributeMethod